client/galaxy/scripts/mvc/tours.js

```diff
@@ -32,10 +32,15 @@ define(['libs/bootstrap-tour'],function(BootstrapTour) {
             }
             if (step.postclick){
                 step.onHide = function(){
-                    _.each(step.postclick, function(postclick){
```

Comment:
Yep, I agree, probably makes the most sense to replace it completely for now.  If we do have need to have a non-onNext triggered event, we can always add it in at that point.